pkg/api/bzz_test.go

```diff
@@ -34,9 +34,9 @@ import (
 
 func TestBzzFiles(t *testing.T) {
 	var (
-		fileUploadResource   = "/bzz"
+		fileUploadResource   = "/v1/bzz"
```

Comment:
in the current implementation, both schemes are supported (you can call either `/bzz` or `/v1/bzz`). i would suggest to keep it this way

Reply:
so some of our unit tests does not test if /v1 works, so I just changed one of them to test it. If I had removed the /v1 routing, the beekeper tests would all be broken.